src/server/qgsserversettings.cpp

```diff
@@ -162,6 +162,27 @@ void QgsServerSettings::initSettings()
                                       };
   mSettings[ sShowGroupSeparator.envVar ] = sShowGroupSeparator;
 
+  // max height
+  const Setting sMaxHeight = { QgsServerSettingsEnv::QGIS_SERVER_WMS_MAX_HEIGHT,
+                               QgsServerSettingsEnv::DEFAULT_VALUE,
+                               "Maximum height for a WMS request. The most conservative between this and the project one is used",
```

Comment:
```suggestion
                               "Maximum height for a WMS request. The lower one of this and the project configuration is used.",
```

I think this would be a bit clearer / easier to read?